core/src/test/java/org/web3j/protocol/core/JsonRpc2_0WebSocketClientJTest.java

```diff
@@ -0,0 +1,40 @@
+package org.web3j.protocol.core;
+
+import java.io.IOException;
+import java.util.concurrent.ScheduledExecutorService;
+
+import org.junit.Test;
+
+import org.web3j.protocol.Web3j;
+import org.web3j.protocol.Web3jService;
+
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class JsonRpc2_0WebSocketClientJTest {
+
+    private ScheduledExecutorService scheduledExecutorService
+            = mock(ScheduledExecutorService.class);
+    private Web3jService service = mock(Web3jService.class);
+
+    private Web3j web3j = Web3j.build(service, 10, scheduledExecutorService);
+
+    @Test
+    public void testStopExecutorOnShutdown() throws Exception {
```

Comment:
How come this test was moved to the websocket test class?

Reply:
Probably renamed this class accidentally in IDEA. Will revert this change.

Reply:
Undone :)